### ISSUE TITLE
MissingPropertyException for "it" in closures

### ISSUE THREAD

**Issue by shrtminded** [jenkinsci/groovy-sandbox#23]
The code below throws a MissingPropertyException when using it inside of closures after version 1.5.

Example 1:

``` groovy
log = ''
(1..10).each{log += it}
```

Example 2:

``` groovy
list =["foo", "bar"]
index = list.findIndexOf{ it == "foo"}
```

**Comment by jglick** [comment]
Already filed as [JENKINS-33468](https://issues.jenkins-ci.org/browse/JENKINS-33468).

**Comment by dusanskopik** [comment]
Hi guys, Is there any fundamental issue preventing fixing this? Any plan to fix it in near future? 
Thanks.

**Comment by frezbo** [comment]
I cannot use Jenkinsfile from SCM and I am blocked due to this

**Comment by jglick** [comment]
> Is there any fundamental issue preventing fixing this?

Unknown until someone tries to fix it.

> Any plan to fix it in near future?

Not currently. Feel free to upvote the JIRA issue or file a PR.

> I am blocked due to this

Unlikely. There is a trivial workaround, as described in JIRA.

**Comment by gerritjvv** [comment]
https://github.com/kohsuke/groovy-sandbox/pull/34/commits/1fffecf0e53f3901559acf707e7e4f766cd33d0d  

This change adds support for implicit "it"

**Comment by gerritjvv** [comment]
@jglick could you have a look at my PR submitted for this fix? thanks.

**Comment by jglick** [comment]
Refiled as https://github.com/jenkinsci/groovy-sandbox/pull/38 by @abayer.

**Comment by dwnusbaum** [comment]
Going by Git tags, looked like the fix for this was released in version 1.14.
